src/dateutils.ts

```diff
@@ -58,6 +58,15 @@ export function month(date: XDate) {
   return fromTo(firstDay, lastDay);
 }
 
+function weeks(firstDate: XDate, numberOfWeeks: number) {
```

Comment:
Little comment about naming, why the 'first' prefix on the param and the consts (can't it just be 'date' or 'startDate', 'year' etc)? Especially when you eventually assign it to 'lastDate' const... (which should rename 'endDate').

Reply:
I can change it from start and end, instead of first and last. The thought here is that this returns a list of dates, given the `numberOfWeeks` argument. I can add a comment here making this more clear as well.